### ISSUE TITLE
Minimap is not showing when using Look Around

### ISSUE THREAD

**Issue by KurzedMetal** [CleverRaven/Cataclysm-DDA#34966]
# Describe the bug
When using Classic sidebar layout (haven't tried others) and the minimap is on the top side

# Steps To Reproduce

1. Pick "classic" sidebar layout
2. Move the minimap at the very top.
3. Press "x" to Look Around
4. Press "N" to activate minimap (if needed)

# Expected behavior
Excepted result: See the minimap on the top side.
Actual result: The sidebar makes space for the minimap at the bottom of the sidebar, but still don't show it.

# Screenshots
My sidebar config:
![image](https://user-images.githubusercontent.com/1426680/67362737-0ca07600-f542-11e9-96f0-aa28b729345f.png)

Result if minimap is on the top:
![image](https://user-images.githubusercontent.com/1426680/67362753-1924ce80-f542-11e9-8314-88b428b0cf3f.png)

It works as expected only if the map is at the bottom:
![image](https://user-images.githubusercontent.com/1426680/67362772-2d68cb80-f542-11e9-9b1d-6a6bef57c171.png)

# Versions and configuration

- OS: Windows
    - OS Version: 10.0 1903
- Game Version: 0.D-8720-gf7e743c [64-bit]
- Graphics Version: Tiles
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    Alternative Map Key [alt_map_key],
    Mutant NPCs [mutant_npcs]
]

**Comment by Night-Pryanik** [comment]
Yeah, this is a known limitation. I'm afraid there is no satisfying solution to this issue, especially if minimap is located not on top or bottom, but somewhere in between.

**Comment by wapcaplet** [comment]
Still reproducible in 0.E-9603-g42468f6cab. In fact, even with the minimap at the bottom, while in the `x` look around screen, pressing `N` causes the side panel to become full-height, obscuring the minimap. Pressing `N` again does not reveal the minimap again.

**Comment by AideeAmei** [comment]
Still reproducible in 286cff2 (tiles)
